backend/db/migrations/20200710060151-alterShopDeployment.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+module.exports = {
+  up: queryInterface => {
+    return queryInterface.sequelize.query(`
+      ALTER TABLE shop_deployments RENAME COLUMN ipfs_gateway TO ipfs_pinner;
+    `)
+  },
+  down: queryInterface => {
+    if (queryInterface.sequelize.options.dialect === 'sqlite') return Promise.resolve()
+
+    return queryInterface.sequelize.query(`
+      ALTER TABLE shop_deployments RENAME COLUMN ipfs_pinner TO ipfs_gateway;
```

Comment:
Is this accurate?  All of the entries in master and staging are, in fact, a gateway URL: Specifically: `https://gateway.pinata.cloud`

Reply:
Yes currently the code populates the gateway URL. After this change is deployed to prod I'm planning on updating the legacy entries to point to the pinner URL so that the data is consistent.